src/test/resources/org/jvnet/hudson/plugins/platformlabeler/generate_redhat_release_output.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Couldn't the existing generate_os_release_output.sh file be renamed and extended to be the `generate_release_output.sh` so that the common code between the two files would be included in a single location?

I think that would also be a benefit on those Red Hat derived operating systems like CentOS which also include an /etc/os-release file.  I'd really like the source data from /etc/os-release and /etc/redhat-release from any platform that provides those files.

Reply:
Yes it probably could, I'll look into combining these.